src/rgw/rgw_op.h

```diff
@@ -1339,6 +1339,16 @@ class RGWDeleteMultiObj : public RGWOp {
   virtual uint32_t op_mask() { return RGW_OP_TYPE_DELETE; }
 };
 
+class RGWInfo: public RGWOp {
+public:
+  RGWInfo() {}
```

Comment:
This constructor is equivalent of the default one. If we really want to be explicit here, I would suggest rather `RGWInfo() = default;`.